code/controllers/verbs.dm

```diff
@@ -20,7 +20,7 @@
 	message_admins("Admin [key_name_admin(usr)] has restarted the [controller] controller.")
 
 /client/proc/debug_controller(controller in list("failsafe", "Master", "Ticker", "Air", "Jobs", "Sun", "Radio", "Configuration", "pAI",
-	"Cameras", "Garbage", "Event", "Alarm", "Nano", "Vote", "Fires",
+	"Cameras", "Garbage", "Event", "Nano", "Vote", "Fires",
```

Comment:
(Unrelated to this PR) I question why all of these exist when you can just click the SS now.